assets/js/script.js

```diff
@@ -0,0 +1,107 @@
+import { UI } from "./ui.js";
+const http = new Http();
+
+const fetchDataFromApi = {
+  getData() {
+    return http.get("products").then((data) => data);
```

Comment:
The additional `then(data => data)` looks redundant, you should be able to remove it. 😉